src/browser/main.js

```diff
@@ -25,13 +26,16 @@ window.Promise = Bluebird;
 // Warnings are useful for user code, but annoying for third party libraries.
 Bluebird.config({ warnings: false });
 
+const initialState = () => window.__INITIAL_STATE__;
```

Comment:
I would refactor out whole registration with react native in mind. Also, the store should be passed in to optionally get app state.